tools/interactor-system-func-calls/src/system_sc_interact_cli.rs

```diff
@@ -0,0 +1,286 @@
+use clap::{Args, Parser, Subcommand};
+use multiversx_sc_snippets::{imports::RustBigUint, multiversx_sc::proxy_imports::*};
```

Comment:
`multiversx_sc_snippets::imports::*` should be enough

Reply:
We need the proxy_imports::TopEncode for NftDummyAttributes